patches/apply-patches.js

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
It would be nice to include this file in tsconfig.dev.json so it at least gets type checked

Reply:
Done 👍